cmd/explaintest/r/explain_indexmerge.result

```diff
@@ -8,61 +8,61 @@ explain select * from t where a < 50 or b < 50;
-└─TableScan_10	98.00	cop[tikv]	table:t, keep order:false
+├─TableRangeScan_8	49.00	cop[tikv]	table:t, range:[-inf,50), keep order:false
+├─IndexRangeScan_9	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
+└─TableFullScan_10	98.00	cop[tikv]	table:t, keep order:false
 explain select * from t where (a < 50 or b < 50) and f > 100;
 id	count	task	operator info
 IndexMerge_12	98.00	root	
-├─TableScan_8	49.00	cop[tikv]	table:t, range:[-inf,50), keep order:false
-├─IndexScan_9	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
+├─TableRangeScan_8	49.00	cop[tikv]	table:t, range:[-inf,50), keep order:false
+├─IndexRangeScan_9	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
 └─Selection_11	98.00	cop[tikv]	gt(test.t.f, 100)
-  └─TableScan_10	98.00	cop[tikv]	table:t, keep order:false
+  └─TableFullScan_10	98.00	cop[tikv]	table:t, keep order:false
 explain select * from t where a < 50 or b < 5000000;
 id	count	task	operator info
 TableReader_7	4000000.00	root	data:Selection_6
 └─Selection_6	4000000.00	cop[tikv]	or(lt(test.t.a, 50), lt(test.t.b, 5000000))
-  └─TableScan_5	5000000.00	cop[tikv]	table:t, range:[-inf,+inf], keep order:false
+  └─TableFullScan_5	5000000.00	cop[tikv]	table:t, keep order:false
 explain select * from t where b < 50 or c < 50;
 id	count	task	operator info
 IndexMerge_11	98.00	root	
-├─IndexScan_8	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
-├─IndexScan_9	49.00	cop[tikv]	table:t, index:c, range:[-inf,50), keep order:false
-└─TableScan_10	98.00	cop[tikv]	table:t, keep order:false
+├─IndexRangeScan_8	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
+├─IndexRangeScan_9	49.00	cop[tikv]	table:t, index:c, range:[-inf,50), keep order:false
+└─TableFullScan_10	98.00	cop[tikv]	table:t, keep order:false
 explain select * from t where b < 50 or c < 5000000;
 id	count	task	operator info
 TableReader_7	4000000.00	root	data:Selection_6
 └─Selection_6	4000000.00	cop[tikv]	or(lt(test.t.b, 50), lt(test.t.c, 5000000))
-  └─TableScan_5	5000000.00	cop[tikv]	table:t, range:[-inf,+inf], keep order:false
+  └─TableFullScan_5	5000000.00	cop[tikv]	table:t, keep order:false
 explain select * from t where a < 50 or b < 50 or c < 50;
 id	count	task	operator info
 IndexMerge_12	147.00	root	
-├─TableScan_8	49.00	cop[tikv]	table:t, range:[-inf,50), keep order:false
-├─IndexScan_9	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
-├─IndexScan_10	49.00	cop[tikv]	table:t, index:c, range:[-inf,50), keep order:false
-└─TableScan_11	147.00	cop[tikv]	table:t, keep order:false
+├─TableRangeScan_8	49.00	cop[tikv]	table:t, range:[-inf,50), keep order:false
+├─IndexRangeScan_9	49.00	cop[tikv]	table:t, index:b, range:[-inf,50), keep order:false
+├─IndexRangeScan_10	49.00	cop[tikv]	table:t, index:c, range:[-inf,50), keep order:false
+└─TableFullScan_11	147.00	cop[tikv]	table:t, keep order:false
```

Comment:
It should be `TableRowIDScan`. Besides `IndexLookup`, we also need to consider `IndexMerge`